spotify_player/src/ui/page.rs

```diff
@@ -618,7 +618,15 @@ fn render_artist_context_page_windows(
     let (album_list, n_albums) = {
         let album_items = albums
             .into_iter()
-            .map(|a| (a.name.clone(), false))
+            .map(|a| {
+                (
+                    match state.configs.app_config.album_format {
+                        config::AlbumFormat::Name => a.name.clone(),
+                        config::AlbumFormat::YearName => format!("({1}) {0}", a.name, a.year()),
+                    },
+                    false,
```

Comment:
I prefer `{name} • {year}` for consistency with other display. `name` is placed before `year` as it's the main information.

Reply:
... or better, create some sort of `render_album_table_window` function similar to  `render_track_table_window` to make the two columns.

Reply:
> Thanks @aome510 for the general positive reception. I agree the `•` is best from a consistency point of view.
> 
> Before making the change on the order however, let me share my considerations. I agree name is the main information. However, the length of the album names are vastly different, whereas the year is always only 4 digits. Thus placing the year in front creates a sort of column, which to me looks more clean and allows for much quicker skimming of both name & year. This is convenient because I remember quite some albums by year instead of name. See also the screenshot.
> 
> ![screenshot_2023-12-05-111757](https://private-user-images.githubusercontent.com/677296/288004568-979dfc34-0cc7-41fe-9c03-9d980c628831.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3OTAwMzAsIm5iZiI6MTcwMTc4OTczMCwicGF0aCI6Ii82NzcyOTYvMjg4MDA0NTY4LTk3OWRmYzM0LTBjYzctNDFmZS05YzAzLTlkOTgwYzYyODgzMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwNVQxNTIyMTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01N2VmOTdhZDNmMzQ3ZWJkNDk1ZTlmMTYwYzgzYTQwZjA2Y2RlZTgyYzM0NTFkMjZjNDIxZGZmN2YxY2E1OGQzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.3E-mCR2jjxH76fIOMjiPVWg0jPcyadN2OPovtkQEd-Q)
> 
> Alternatively, if you indeed prefer the year after the name, would you be ok with right aligning the year by left padding it? It would be a bit more elaborate than the current implementation, but reading the code the necessary pad could be calculated from `related_artists_rect.width`.

What you said makes sense. I'm also good with `{year} • {name}` as it looks better. I will try to avoid adding more complexity to implement the year alignment with the `{name} • {year}` format.

Reply:
Perfect! I just removed the config option and left `{year} • {name}` in place. Note that the year is still appended in brackets in the places where the Album `fmt` is used. E.g. in the Albums list of the Library page (see model.rs).